.woodpecker.yml

```diff
@@ -35,17 +41,23 @@ steps:
       - cargo clippy --all-targets --all-features --
         -D warnings -D deprecated -D clippy::perf -D clippy::complexity
         -D clippy::dbg_macro
+    when:
+      - event: pull_request
 
   cargo_test:
     image: *rust_image
     environment:
       CARGO_HOME: .cargo
     commands:
       - cargo test --all-features --no-fail-fast
+    when:
+      - event: pull_request
 
   cargo_run:
     image: *rust_image
     environment:
       CARGO_HOME: .cargo
     commands:
       - cargo run
```

Comment:
Please change this to `cargo run -- --max-crawl-distance 0` so it doesnt spend 10 minutes crawling all Lemmy instances.

Reply:
True clippy and test is enough. Then you can remove this and merge afterwards.